synse/commands/fan_sensors.py

```diff
@@ -1,11 +1,59 @@
 """Command handler for the `fan sensor` route.
 """
 
+import datetime
+from collections import OrderedDict
+
 from synse import cache
 from synse.commands.read import read
 from synse.log import logger
 
 
+# TODO: Need a note in the configuration files about auto_fan relying on
+# device information names in order to find these sensors.
+def _translate_device_info(device_info):
```

Comment:
made note of this in a different PR as well, just saying here again since its pertinent. 

I'm not a fan of using the device 'info' field to identify a device. if we just want to ensure its a unique device, it has a unique id that can be used. an issue for a different day, but I think there are some better/cleaner solutions we could have for identifying/labeling devices that we want to use.

since this is work to get autofan working again, it'll be fine for now, but we'll eventually need to think through a better design for things like this.